index.js

```diff
@@ -0,0 +1,33 @@
+mapboxgl.accessToken = 'pk.eyJ1IjoibHVpc3NlIiwiYSI6ImNpdGoyZ3Q1bjA2eGoybm8zdjFrN2RuMmQifQ.xiEfhxmHUIKN3T3frogq4w';
```

Comment:
This is the kind of thing you probably _don't_ want to check into GitHub.

https://github.com/startup-systems/mashup#tips